internal/cli/server_upgrade.go

```diff
@@ -586,5 +606,11 @@ Please run the following commands if you wish to unset these runner profiles
 from being the default:
 
 %[2]s
+`)
+
+	runnerProfileUpgradeConfigError = strings.TrimSpace(`
+The plugin config for runner profile %[1]s is incorrect. Please run
+waypoint runner profile set -name=%[2]s -plugin-config=<path_to_config_file>
+with the corrected plugin configuration to fix this.
```

Comment:
```suggestion
The plugin config for runner profile %[1]s failed to correctly be parsed.
Plugin Config Err: %[3]s

Please run the following with the corrected plugin configuration to fix this.

waypoint runner profile set -name=%[2]s -plugin-config=<path_to_config_file>
```

We should include the error here too, which I think will show the "validated" reason for why the config isn't correct. Something like this might come from the parser.... `"'Requested' was not expected...."`

We don't have to do it in this PR but if we can it would be great to fix these docs too, they don't have the right type and need to be updated to include the nested block fields 😭 That might make this easier for people to figure out how to set the right config: https://tip.waypointproject.io/plugins/kubernetes#required-parameters-4